saleor/graphql/csv/tests/mutations/test_export_products.py

```diff
@@ -203,6 +204,53 @@ def test_export_products_mutation_ids_scope(
     ).exists()
 
 
+@patch("saleor.graphql.csv.mutations.export_products_task.delay")
+def test_export_products_mutation_ids_scope_invalid_object_type(
+    export_products_mock,
+    staff_api_client,
+    product_list,
+    permission_manage_products,
+    permission_manage_apps,
+):
+    query = EXPORT_PRODUCTS_MUTATION
+
+    products = product_list[:2]
+
+    ids = []
+    pks = set()
+    for product in products:
+        pks.add(str(product.pk))
+        ids.append(graphene.Node.to_global_id("ProductVariant", product.pk))
```

Comment:
```suggestion
    ids = [graphene.Node.to_global_id("ProductVariant", product.pk) for product in products]
```
I don't see `pks` set used anywhere?